CustomVideoRenderer/VideoRenderer.cs

```diff
@@ -0,0 +1,126 @@
+﻿using OpenTok;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+
+namespace CustomVideoRenderer
+{
+    /// <summary>
+    /// Follow steps 1a or 1b and then 2 to use this custom control in a XAML file.
+    ///
```

Comment:
Maybe we should include these instructions in the README file for the sample app (not in code comments).

Reply:
This is something is added automatically by the IDE when you create a WPF Control